src/test/common/clara/test_dsl.cljc

```diff
@@ -337,7 +337,6 @@
                           {:r :r4
                            :v :good}])
             (->> (-> empty-session
-                     (insert ^{:type :test} {})
```

Comment:
I propose fixing this by changing the fact type inserted here to something other than :test rather than eliminating the LHS of the rule.  My reasoning is that a test for shadowing seems a bit stronger if we force the evaluation to wait until the insert call is made (and not at any time after the call to create the session).  Not a big deal, but it is also an easy change.

Reply:
can do